### DISCUSSION TITLE
Guidance for RemoteApp

### DISCUSSION THREAD

**Discussion by necouchman** [neutrinolabs/xrdp#3071] [Q&A] [answered]
According to the Wiki (https://github.com/neutrinolabs/xrdp/wiki/RAIL-support-in-xrdp), basic support for RemoteApp (RAIL) exists in xrdp. I'm running xrdp on Rocky9, and I see the channel available, but when I specify the full path to an application using the client-side Remote App commands, I'm unable to get it to work. Is there any guidance on how to use the basic RemoteApp support, incomplete though it may be?

I see a previous discussion from July of last year on the topic with a simple note that support doesn't exist, but the wiki seems to (partially?) contradict that.

**Comment by necouchman** [comment]
Guessing this hasn't changed since #3020.

**Reply by matt335672** [reply, marked as answer]
We haven't made any progress on this lately.

**Reply by necouchman** [reply]
Thanks @matt335672, appreciate the update.
